internal/node/node.go

```diff
@@ -230,7 +230,10 @@ func (n *Node) init() error {
 			n.log.Error("Failed to load node configuration file", slog.Any("err", _err), slog.String("config_path", n.nodeOpts.ConfigFilepath))
 			err = errors.Join(err, _err)
 		} else {
-			n.log.Info("Loaded node configuration", slog.String("config_path", n.nodeOpts.ConfigFilepath))
+			n.log.Info("Loaded node configuration",
+				slog.String("config_path", n.nodeOpts.ConfigFilepath),
+				slog.Any("deny_trigger_subjects", n.config.DenyTriggerSubjects),
```

Comment:
This `deny_trigger_subjects` seemed a little out of place here.

Reply:
Hehe it's leftover debug. I didn't see it when cleaning because it wasn't a printf ;)